pkg/visor/init.go

```diff
@@ -257,6 +259,7 @@ func initSctpClient(ctx context.Context, v *Visor, log *logging.Logger) error {
 }
 
 func initDmsgCtrl(ctx context.Context, v *Visor, log *logging.Logger) error {
+	<-v.isNetConf
```

Comment:
You don't need this synchronization. Since dmsgCtrl module depends on snet module, `initDmsgCtrl` is guaranteed to run after `initSnet`. 

After snet refactoring (the changes just made it to develop), snet module and snet.Network are removed, and dmsg is taken out into its own module and dmsgCtrl depends on dmsg directly.

Reply:
Yes merging with develop rn, will make the changes.